compiler/tests-jsoo/dune

```diff
@@ -16,22 +16,32 @@
  (enabled_if
   (>= %{ocaml_version} 5.1.1))
  (inline_tests
-  (modes js best))
+  (modes js wasm best))
+ (preprocess
+  (pps ppx_expect)))
+
+(library
+ (name jsoo_testsuite_parsing)
+ (modules test_parsing calc_parser calc_lexer)
+ (libraries unix compiler-libs.common js_of_ocaml-compiler)
+ (enabled_if
+  (>= %{ocaml_version} 5.1.1))
```

Comment:
Why is this conditioned on ocaml >= 5.1.1 ?

Reply:
Because this PR landed on OCaml 5.1.1: https://github.com/ocaml/ocaml/pull/12046

Reply:
Ok, so this is do that we can drop the [flush_stubs.c](https://github.com/ocsigen/js_of_ocaml/pull/1724/files/c5d3694429833fd137748ee8230f7efd18962fc7#diff-1d121ef25538fe6d01a2ad0a42e4b26cf7f92b3af30a594882ebeebf679c0096)